mobile-app/lib/ui/views/podcast/widgets/podcast_title_widget.dart

```diff
@@ -185,214 +188,223 @@ class PodcastTileState extends State<PodcastTile> {
     }
   }
 
-  String _parseDuration(Duration dur, BuildContext context) {
-    String hours = (widget.episode.duration!.inMinutes ~/ 60).toString();
-    String minutes = (widget.episode.duration!.inMinutes % 60).toString();
-
-    if (dur.inMinutes > 59) {
-      return context.t.podcast_duration_hours(
-        hours,
-        minutes,
-      );
-    } else {
-      return context.t.podcast_duration_minutes(
-        minutes,
-      );
-    }
+  String getFirstSentenceFromHtml(String? html) {
+    if (html == null || html.isEmpty) return '';
+    final document = html_parser.parse(html);
+    final text = document.body?.text ?? '';
+    final match = RegExp(r'([^.?!]*[.?!])').firstMatch(text);
+    return match != null ? match.group(0)!.trim() : text.trim();
```

Comment:
This will make each episode tile have inconsistent height. If you browse through some of them, you'll see that the first sentence can either take more than 5 lines and other times hardly 1 line. Instead of doing this we keep the current behavior ie. show the first 3 lines and truncate it with ellipsis (`...`) at the end